lib/bamboo/adapters/mandrill_adapter.ex

```diff
@@ -103,12 +103,23 @@ defmodule Bamboo.MandrillAdapter do
       subject: email.subject,
       text: email.text_body,
       html: email.html_body,
-      headers: email.headers,
-      attachments: attachments(email)
+      headers: email.headers
     }
+    |> add_attachments(email)
     |> add_message_params(email)
   end
 
+  defp add_attachments(mandrill_message, %{attachments: attachments}) do
+    {files, images} =
+      attachments
+      |> Enum.reverse()
+      |> Enum.split_with(&is_nil(&1.content_id))
+
+    mandrill_message
+    |> Map.put(:attachments, format_attachments(files))
+    |> Map.put(:images, format_attachments(images))
```

Comment:
Do you know why we're calling this `images`? Will inline attachments always be images or is that how mandrill categorizes inline attachments?

Reply:
Yes, of course! https://mailchimp.com/developer/transactional/api/messages/send-new-message/
Inside message attribute at the end.

Is similar to attachments with the difference of send the cid instead of the filename.

Reply:
Thanks for sending that link. That's very helpful! This is the section I'm looking at: 

![Screen Shot 2021-06-11 at 8 56 45 AM](https://user-images.githubusercontent.com/3245976/121690889-64338280-ca94-11eb-81cf-88320c545999.png)


Based on what I'm seeing, the changes in this PR seem good. 

My only lingering question is: do you know if it's possible to add non-image files as inline attachments? 

- If that's the case, then someone could add a non-image attachment with `content-id` but we would _incorrectly_ put it in the `:images` field. 
- If Mandrill only allows for images as inline attachments, then I think this PR is good to go.

Reply:
I added a new commit to check the content_type too before add it as inline image.